relay/sources/relayd/src/data/report.rs

```diff
@@ -594,6 +682,41 @@ mod tests {
             maybe_report(report).unwrap().1,
             Err("2018-08-24T15:55:01+00:00 R: @@Common@@broken".to_string())
         );
+        let report = "garbage\n2018-08-24T15:55:01+00:00 R: @@Common@@result_repaired@@hasPolicyServer-root@@common-root@@0@@CRON Daemon@@multi\r\n2018-08-24T15:55:01+00:00 line@@2018-08-24 15:55:01 +00:00##root@#Cron daemon status was repaired\r\n";
```

Comment:
Why is there a date in the after the newline in the component key? On windows node we do no report like that. The date prefix is only added on new report lines. Not on newline inside a report.

Reply:
Because on Linux the timestamp do not come from the logs but are added by the wrapper, so we can't really know which line require a prefix, so we add it on all lines.

Reply:
I added another test for the full windows case.